recipes/igwn-lldd-fdio/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "igwn-lldd-fdio" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: 862fcb1f7f7039ebe126e9d532835696fa8b29ef954bca162b687dfccdbad8d9
+
+build:
+  entry_points:
+    - kafka2fdio = igwn_lldd_common.kafka2fdio:main
+    - fdio2kafka = igwn_lldd_common.fdio2kafka:main
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vvv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools >=38.2.5
+  run:
+    - configargparse
+    - igwn-lldd-common
+    - pyfd
+    - python >=3.6
+    - pythonvirgotools
+
+test:
+  requires:
+    - pip
+    - pytest
+    - flaky
+  commands:
+    # check metadata
+    - python -m pip check igwn-lldd-fdio
+    - python -m pip show igwn-lldd-fdio
+    # run test suite
+    - python -m pytest -ra -v --pyargs igwn_lldd_fdio.tests
+    # check entry points
+    - python -m igwn_lldd_common.fdio2kafka --help
+    - python -m igwn_lldd_common.kafka2fdio --help
```

As above (I think the library name is wrong anyway):

```suggestion
    - fdio2kafka --help
    - kafka2fdio --help
```